Radio/example.jsx

```diff
@@ -99,6 +105,13 @@ export default {
         value='ipsum'
       />,
 
+      'Partially collapsed & controlled': <Radio
+        options={optionsWithContent}
+        expandLabel='Show me all the options'
+        collapsedOptions={2}
```

Comment:
Deadline Driven Development. I've heard on the internets that DDD is a good thing, so 👍